collectors/pod.go

```diff
@@ -324,13 +338,22 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 				c.Name, nodeName)
 		}
 
+		if gpu, ok := req[v1.ResourceNvidiaGPU]; ok {
```

Comment:
It is ok for me to remove "gpu resource request metric", it is not a big deal.
Actually u can see from the code, the doc maybe some kinda wrong. It is ok to have request as long as it is same with limit.
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/validation/validation.go#L3717